src/context/SettingsContext/SettingsContextProvider.tsx

```diff
@@ -172,17 +170,33 @@ export function SettingsContextProvider({ children }: { children: ReactNode }) {
   const [externalSwapsEnabled, setExternalSwapsEnabled] = useLocalStorageByChainId(
     chainId,
     EXTERNAL_SWAPS_ENABLED_KEY,
-    false
+    true
   );
   const [debugSwapMarketsConfig, setDebugSwapMarketsConfig] = useLocalStorageSerializeKey<
     undefined | { disabledSwapMarkets?: string[]; manualPath?: string[] }
   >([chainId, DEBUG_SWAP_MARKETS_CONFIG_KEY], undefined);
 
+  const [externalSwapsWasForceEnabled, setExternalSwapsWasForceEnabled] = useLocalStorageByChainId(
+    chainId,
+    EXTERNAL_SWAPS_ENABLED_KEY,
+    false
+  );
+
+  useEffect(() => {
+    // Force enable external swaps for Botanix
```

Comment:
hm, then maybe just use a condition like if botanix || externalSwapsEnabled and don't modify the state?